.travis.yml

```diff
@@ -109,7 +109,7 @@ matrix:
         - os: linux
           env: SETUP_CMD='test --remote-data=astropy --readonly'
                LC_CTYPE=C.ascii LC_ALL=C
-               PYTEST_VERSION=3.7
+               PYTEST_VERSION=4.4
```

Comment:
There may be a slight chance that in a half year we won't remember why this is set to 4.4, but anyway, we can always find the way with git blame back to this PR and see the comments that it's 4.4 as otherwise travis picked up an old version that was incompatible with pytest-xdist.

Reply:
I don't remember why we pin the whole build to 3.10 either, but that is a battle for another time. 😅

Reply:
There must have been issues with pytest 4.0. I suppose.